### PR TITLE
Buffs cyborg RCD

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
It literally takes over 1k charge to make a single wall right now, how awful.

### CODE REVIEW

**Comment by lzimann** [comment]
:+1:

File: code/game/objects/items/weapons/RCD.dm [hunk 1]
```diff
@@ -475,12 +475,12 @@ RCD
...
 	if(!isrobot(user))
 		return 0
-	return user:cell:use(amount * 72) //borgs get 1.3x the use of their RCDs
+	return user:cell:use(amount * 10) //borgs get 1.3x the use of their RCDs
 
 /obj/item/weapon/rcd/borg/checkResource(amount, mob/user)
 	if(!isrobot(user))
 		return 0
-	return user:cell:charge >= (amount * 72)
+	return user:cell:charge >= (amount * 10)
```
Comment (neersighted) [inline code/game/objects/items/weapons/RCD.dm]: Colon cancer, typecast that shit

**Comment by GunHog** [comment]
<b>THANK. YOU.</b>

**Comment by xxalpha** [comment]
Suspicious.

**Comment by Lumipharon** [comment]
How man airlocks can engie borgs make after thiss per recharge?

**Comment by Dorsisdwarf** [comment]
As it is, airlocks use 5,000 charge to decon/create

After this, they will use 7.2 times less charge.

**Comment by Lumipharon** [comment]
Whelp. Enjoy cutting through 500 airlocks to get to the AI.

**Comment by Actionb** [comment]
21 airlocks are too much. Something around 2000-3000 charge per airlock (5-7 airlocks with 15k cell) would be good enough for repairs.

**Comment by Dorsisdwarf** [comment]
Actually I misread it the first time. I think airlocks are only 16 to build, but 30 to decon.

This means that building airlocks went from ~1100 charge to ~160

aaaaaaaaaaaaaaaaaa

File: code/game/objects/items/weapons/RCD.dm [hunk 2]
```diff
@@ -475,12 +475,12 @@ RCD
 /obj/item/weapon/rcd/borg/useResource(amount, mob/user)
 	if(!isrobot(user))
 		return 0
-	return user:cell:use(amount * 72) //borgs get 1.3x the use of their RCDs
+	return user:cell:use(amount * 10) //borgs get 1.3x the use of their RCDs
```
Comment (duncathan) [inline code/game/objects/items/weapons/RCD.dm]: Update comment

**Comment by tkdrg** [comment]
This will never pop up for legit repairs and it's an enormous buff to rogue engiborgs. Just no.

**Comment by KorPhaeron** [comment]
I dont want to repair I want to build things.

If I lock myself into playing something that can literally only build for 30-50 minutes, building shouldnt be incredibly slow and tedious.

**Comment by xxalpha** [comment]
:+1:

**Comment by tkdrg** [comment]
engiborgs are incredibly versatile for a borg and they're pretty much the most efficient way to repair/build short of admin tools. building isn't slow nor is it tedious unless you're making huge projects, and even then it's way easier as an engiborg when compared to literally everything else. This is going to make engiborgs a LOT more effective at barricading places with infinite bolted airlocks of doom (which granted mostly affects traitor AI apart from maybe some odd gang/cult subverted borg, but still). I just don't see a need for this change, and it has plenty of potential to backfire.

**Comment by GunHog** [comment]
How about this? We remove the whole cell-draining mechanic for the Borg RCD, so it is not destroying its charge on trying to fix or build things because of their use for Malf AIs. Give it the same capacity as any other RCD, no more, no less. Have it refill just as every other borg module refills - using the charger.

Keeps things simple and avoids having borgs drain themselves whenever a bomb happens.

**Comment by KorPhaeron** [comment]
What if we made airlocks more expensive for cyborgs so I can still spam floors and walls for my dumb forts

**Comment by xxalpha** [comment]
No GunHog, terrible idea.

**Comment by tkdrg** [comment]
I like kor's idea better honestly.

**Comment by KorPhaeron** [comment]
I don't really wanna be on the hook for balance whining about robots for the next 6 months though

**Pull request closed without merging**